src/App.tsx

```diff
@@ -1,192 +1,71 @@
-          do not offer let alone encourage us to imagine and create alternative
-          futures for the internet. What if everyone who used the internet was
-          encouraged to dream about it? What would it look like if we
-          collectively imagine and build a better internet together?
-        </p>
-        <p>
-          If you have an internet dream you would like to contribute and/or want
-          to help us cultivate this space, please{" "}
-          <a href="mailto:spencerc99@gmail.com,j.zhao2k19@gmail.com?subject=(we)bsite dreams">
-            reach out to us
-          </a>
-          .
-        </p>
-      </div>
-      <div id="desk">
-        {Object.entries(letters).map(([id, letter]) => (
-          <LetterView letter={letter} key={id} id={id} shared={sharedMap} />
-        ))}
-      </div>
-      <footer>
-        (we)bsite is a project by{" "}
-        <a href={people.spencer.url}>{people.spencer.fullName}</a> and{" "}
-        <a href={people.jacky.url}>{people.jacky.fullName}</a>. Poke around{" "}
-        <a href="https://github.com/we-bsite/we-bsite">the source code</a>!
-      </footer>
+      <Dialog.Root>
+        <div className="App">
+          <h1>(we)bsite</h1>
+          <p>
+            <em>(we)bsite</em> is a living collection of internet dreams from
+            people like you, inhabitants of the internet. It aims to create
+            space to hold, show, and uplift everyday visions and hopes for the
+            internet.
+          </p>
+          <p>
+            Despite all the time we spend on the internet, most of us are
+            passive consumers of content and interfaces. Our standard digital
+            environments do not offer let alone encourage us to imagine and
+            create alternative futures for the internet.
+          </p>
+          <p>
+            What if everyone who used the internet was encouraged to dream about
+            it? What would it look like if we collectively imagine and build a
+            better internet together?
+          </p>
+          <p>
+            Change starts with possibility. We hope <em>(we)bsite</em> can be a
+            catalyst for change by showing all the love, care, memories, and
+            dreams that people have for the internet.
+          </p>
+          <p>
+            <em>What do you want from the internet?</em>
+          </p>
+        </div>
+        <Desk />
+        <footer>
+          (we)bsite is a project by{" "}
+          <a href={people.spencer.url}>{people.spencer.fullName}</a> and{" "}
+          <a href={people.jacky.url}>{people.jacky.fullName}</a>. Poke around{" "}
+          <a href="https://github.com/we-bsite/we-bsite">the source code</a>!{" "}
+          <p>
+            if your ears are perked, please{" "}
+            <a href="mailto:spencerc99@gmail.com,j.zhao2k19@gmail.com?subject=(we)bsite dreams">
+              reach out to us
+            </a>
```

Comment:
hm I wonder if this extra line makes the footer too long

Reply:
mmm maybe i put it on same line?